jwst/resample/resample_step.py

```diff
@@ -33,10 +33,10 @@ class ResampleStep(Step):
     class_alias = "resample"
 
     spec = """
-        pixfrac = float(default=1.0)
-        kernel = string(default='square')
-        fillval = string(default='INDEF')
-        weight_type = option('ivm', 'exptime', default='ivm')
+        pixfrac = float(default=None)
+        kernel = string(default=None)
+        fillval = string(default=None)
+        weight_type = option('ivm', 'exptime', None,default=None)
```

Comment:
I need a whitespace before 'default' ? That is not how the other parameters are set up.

And yes I had to add the extra option of None because when the default is None the step failed because None was not a valid option.